data_pipeline/data_processing/aggregatedcsv2redis.py

```diff
@@ -0,0 +1,87 @@
+"""
+Do not merge. For discussion only.
```

Comment:
Update docstring? Perhaps this could link to https://docs.google.com/document/d/1pcSLUKkdn0aRsr5ml45007-bAVt40E0jdizLT0E-P0Q/edit or could include the sample output you included in the commit log?